### PR TITLE
Improve wording of progress update

### PR DESCRIPTION
**Pull request by adrian-prantl**
Improve wording of progress update assuming that nobody outside of LLDB developers knows what a "compile unit import" is.

### CODE REVIEW

**Comment by adrian-prantl** [comment]
@swift-ci test

File: lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp [hunk 1]
```diff
@@ -8829,9 +8829,9 @@ bool SwiftASTContext::GetCompileUnitImportsImpl(
   if (cu_imports.size() == 0)
     return true;
 
-  Progress progress("Getting Swift compile unit imports",
-                    compile_unit->GetPrimaryFile().GetFilename().GetCString(),
-                    cu_imports.size());
+  std::string category = "Importing Swift module dependencies for ";
+  category += compile_unit->GetPrimaryFile().GetFilename();
```
Comment (chelcassanova) [inline lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp:8833]: Would `main.swift` always be the primary filename here? The category would be "Importing Swift module dependencies for main.swift" which I think works as a category name (as opposed to "Importing Swift module dependencies for"), I just wanna make sure that the filename here wouldn't change.

Comment (adrian-prantl) [reply lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp:8833]: The idea is we're debugging `main.swift`, which contains
```
import Foundation
import Cocoa
```
and then we should get:
```
Importing Swift module dependencies for main.swift: Foundation
Importing Swift module dependencies for main.swift: Cocoa
```

Comment (adrian-prantl) [reply lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp:8833]: So, in short: yes!

**Review by JDevlieghere — APPROVED** [review]

File: lldb/test/API/functionalities/progress_reporting/swift_progress_reporting/TestSwiftProgressReporting.py [hunk 2]
```diff
@@ -38,14 +38,16 @@ def test_swift_progress_report(self):
             "Loading Swift module",
             "Importing modules used in expression",
             "Setting up Swift reflection",
-            "Getting Swift compile unit imports",
+            "Importing Swift module dependencies for main.swift",
```
Comment (medismailben) [inline lldb/test/API/functionalities/progress_reporting/swift_progress_reporting/TestSwiftProgressReporting.py:41]: nit: We're only doing partial string matching on the test so we could have drop the file name at the end.

**Pull request merged**